cli/ssync/main.go

```diff
@@ -58,16 +57,15 @@ Examples:
 
 		endpoint := sparse.TCPEndPoint{Host: *host, Port: int16(*port)}
 		if *verbose {
-			log.SetLevel(defaultVerboseLogLevel)
+			log.SetLevel(log.DebugLevel)
```

Comment:
This verbose check should be outside, also, don't use fmt.Fprintf(), use log instead.